tests/lm_eval/conftest.py

```
@@ -705,3 +710,19 @@ def lmevaljob_gpu(
         ],
     ) as lmevaljob:
         yield lmevaljob
+
+
+# Shared fixtures from ai_safety
+@pytest.fixture(scope="session")
+def trustyai_operator_configmap(
+    admin_client: DynamicClient,
+) -> ConfigMap:
```

Flake8 (7.3.0)

[error] undefined name 'ConfigMap'

(F821)